packages/asset/test/fixtures/catalystFixture.ts

```diff
@@ -0,0 +1,81 @@
+import { deployments, getUnnamedAccounts } from "hardhat";
+import {
+  CATALYST_BASE_URI,
+  CATALYST_IPFS_CID_PER_TIER,
+  CATALYST_DEFAULT_ROYALTY,
+} from "../../constants";
+
+export const runCatalystSetup = deployments.createFixture(
+  async ({ deployments, getNamedAccounts, ethers }) => {
```

Comment:
I think we shouldn't need to use `deployments` at all now, can you check the example again pls? The idea is that we don't need hardhat-deploy inside packages that aren't `packages/deploy`

Reply:
Had a problem with that we are using a upgradable Catalyst which disables initialization on the implementation contract so to deploy a proxy and the catalyst implementation I used the hardhat-deploy